hello.html

```diff
@@ -1,107 +1,118 @@
-         <h3 class="author">By Evgeny Khrolenok & Igor Mikheiko</h3>
-         <p class="main_topic">It wasn’t long after Hollywood released its first 3D films that the movie format quickly gained huge popularity worldwide. Thanks to developments in video-recording technology, any user can now shoot a video on their own. You can make a stereo record of memorable events in your life or create wonderful material for your business.</p>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Abishek">
+    <title>
+        The Content Teacher
+    </title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Droid+Sans" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+</head>
 
-         <img src="https://www.smashingmagazine.com/wp-content/uploads/2017/07/shooting-process-500w-opt.jpg" width="500" height="284&quot;&quot;" class="topic_img" alt="How We Built An iOS App To Shoot A 3D Video (Case Study)">
-       
-         <p class="sub_topic"> Our team was also attracted to 3D filming. We thoroughly studied the features of the human visual apparatus and the technical details of stereoscopic photography. Then, we decided to develop an iOS app to shoot 3D videos and upload the videos to YouTube. The idea behind the app was to facilitate the shooting of 3D video by mounting two iPhones to a special frame — and we did it! That was how the Stereo Video Recorder app appeared.
- 
-         </p>
-         <button class="read_btn">Read More...</button>
-       </div>
-       <div class="child">
-         <h2 class="sub_heading">Building Pattern Libraries With Shadow DOM In Markdown</h2>
-         <h3 class="author">By Heydon Pickering</h3>
-         <p class="main_topic">Some people hate writing documentation, and others just hate writing. I happen to love writing; otherwise, you wouldn't be reading this. It helps that I love writing because, as a design consultant offering professional guidance, writing is a big part of what I do. But I hate, hate, hate word processors.</p>
+<body>
+    <header>
+        <div class="container">
+            <img class="logo" src="https://engineering.dartmouth.edu/styleguide/web/png/shield/ThayerShield_Green.png">
+            <h1 class="main_heading"> The Content Teacher </h1>
+        </div>
+    </header>
+    <nav class="display_flex">
+        <div class="container">
+            <ul id="un_list">
+                <li class="dropbtn active">
+                    <a id="home_btn" href="hello.html"> Home</a>
+                </li>
+                <li class="dropdown dropbtn">Information
+                    <div class="dropdown-content">
```

Comment:
For sub-menu items use another <ul><li> inside that particular <li> or use nested list